specifyweb/frontend/js_src/lib/components/DataModel/__tests__/businessRules.test.ts

```diff
@@ -210,6 +210,48 @@ describe('Collection Object business rules', () => {
   });
 });
 
+describe('CollectionObjectGroup business rules', () => {
+  const getBaseCOG = () => {
```

Comment:
for consistent casing

```suggestion
  const getBaseCog = () => {
```